packages/help-center/src/components/help-center-article.tsx

```diff
@@ -53,6 +55,66 @@ export const HelpCenterArticle = () => {
 		}
 	}, [ post, query, sectionName ] );
 
+	const toggleTab = ( element: Element, show: boolean ) => {
```

Comment:
Would it be possible to extract the tabbing functionality in a separate hook that we can call here?

Reply:
🤔 Let me check how I can refactor this